Sources/SnapshotTesting/Common/View.swift

```diff
@@ -772,6 +841,106 @@
             )
           }
         }
+        
+        public static func iPhone15(_ orientation: ViewImageConfig.Orientation) -> UITraitCollection {
+            let base: [UITraitCollection] = [
+              .init(forceTouchCapability: .available),
+              .init(layoutDirection: .leftToRight),
+              .init(preferredContentSizeCategory: .medium),
+              .init(userInterfaceIdiom: .phone)
+            ]
+            switch orientation {
+            case .landscape:
+              return .init(
+                traitsFrom: base + [
+                  .init(horizontalSizeClass: .regular),
+                  .init(verticalSizeClass: .compact)
+                ]
+              )
+            case .portrait:
+              return .init(
+                traitsFrom: base + [
+                  .init(horizontalSizeClass: .compact),
+                  .init(verticalSizeClass: .regular)
+                ]
+              )
+            }
+          }
+
+          public static func iPhone15Plus(_ orientation: ViewImageConfig.Orientation) -> UITraitCollection {
+            let base: [UITraitCollection] = [
+              .init(forceTouchCapability: .available),
```

Comment:
Not sure these match the actual values, and other places: https://github.com/playbook-ui/playbook-ios/blob/798f686a6d73318185f85d844876274e91f1f3a2/Sources/Playbook/SnapshotSupport/SnapshotDevice.swift#L739

iPhone X series was the last to support force touch afaik

Reply:
Updated

Reply:
Hi @luispadron can you approve it ?

Reply:
I don't have maintainer access unfortunately

Reply:
:( who can approve for merge ?

Reply:
@mbrandonw or @stephencelis